### PR TITLE
Update Links and Logos

### PR DESCRIPTION
**Pull request by mrmundt**
This PR does some quick updates to the main page to reflect newer PSIP logos and URLs (as well as adding the new logos to the assets directory).

I chose to remove Gitter because it's had almost no activity. Happy to add it back in.

### CODE REVIEW

**Comment by mrmundt** [comment]
Screen Shot 2022-06-27 at 08 01 12

We have a logo, so might as well use it!

File: index.md [hunk 1]
```diff
@@ -6,12 +6,11 @@ Scientific software teams are typically focused on the creation of a new set of
 
 The Productivity and Sustainability Improvement Planning (PSIP) process recognizes that productivity and sustainability improvements for scientific software benefit from an incremental, iterative approach.
 
-Get Started with PSIP:
+_Get Started with PSIP_:
```
Comment (charmoniumQ) [inline index.md]:
```suggestion
## Get Started with PSIP
```
This is semantically a header, right?

**Review by charmoniumQ — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (mrmundt) [reply index.md]: Technically, maybe? I like the idea of making it a header more than its current format. So I accept.

**Comment by elaineraybourn** [comment]
I think you should add Gitter back in until we can discuss as a team about whether we want to keep it.

**Pull request merged**